### ISSUE TITLE
`Width: unset` not showing as unsupported by IE

### ISSUE THREAD

**Issue by ZacharyRSmith** [ismay/stylelint-no-unsupported-browser-features#9]
I ran stylelint with `width: unset` as a CSS rule, but the linter did not warn me about this, even though it was warning me about other things not supported by IE.

CanIUse shows `unset` as being unsupported by IE.

**Comment by ismay** [comment]
If you're seeing other warnings then I suspect it's a doiuse issue (which we use to detect browser support). Try using only doiuse (without stylelint) and see if that yields the same results. If it does it's a doiuse issue and you can open an issue there. Otherwise let me know and we'll reopen and investigate further (a reduced testcase would be nice in that case).

**Comment by ZacharyRSmith** [comment]
For future reference:

My workaround:

I'm using [stylelint](https://github.com/stylelint/stylelint) to run doiuse, in addition to other linting.  To warn about "initial" and "unset", I have the below rule in my `.stylelintrc`. The built-in rule 'declaration-property-value-blacklist' blacklists the property-value pair I give it.  The pair I give it is:

property: "/^.*/" (match all regex)
value: ["initial", "unset"]

So, all properties with a value of "initial" or "unset" will throw a violation.

I set the violation severity to warning.

```
		"declaration-property-value-blacklist": [{
			"/^.*/": ["initial", "unset"]
		}, { "severity": "warning" }],
```

**Comment by ismay** [comment]
Thanks @ZacharyRSmith. Also, I'll quote your other comment for others encountering this issue: 

> From what I can tell looking at the [doiuse] source code, no unsupported values are being checked -- only unsupported properties.

So this has to do with doiuse, which is wrapped by this lib. See this issue: https://github.com/anandthakker/doiuse/issues/56
